react/react1/week2/homework/my-app/src/App.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import './App.css';
+import {Header} from './components/Header';
+import { TodoList } from './components/TodoList';
+
+const todos=[
+  {
+    "id": 1,
+    "description": "Get out of bed"
+  },
+  {
+    "id": 2,
+    "description": "Brush teeth"
+  },
+  {
+    "id": 3,
+    "description": "Eat breakfast"
+  }
+]
+function App() { 
+
+
+  
+  return (
+    <div className="App">
+      <Header header='Todo list' />
+      <TodoList 
+      Noitems='No items'
```

Comment:
You are passing Noitems but it doesn't exist in you TodoList component! If you don't need it you should remove it.
Also, naming :)
Noitems -> noItems

Reply:
Thanks 👍 .